src/xo-app/backup/new/index.js

```diff
@@ -561,133 +563,137 @@ export default class New extends Component {
-                    </select>
-                  </fieldset>
-                  {(method === 'vm.rollingDeltaBackup' || method === 'vm.deltaCopy') && <div className='alert alert-warning' role='alert'>
-                    <Icon icon='error' /> {_('backupVersionWarning')}
-                  </div>}
-                  {backupInfo && <div>
-                    <GenericInput
-                      label={<span><Icon icon={backupInfo.icon} /> {_(backupInfo.label)}</span>}
-                      required
-                      schema={backupInfo.schema}
-                      uiSchema={backupInfo.uiSchema}
-                      onChange={this.linkState('mainParams')}
-                      value={this._getMainParams()}
-                    />
+        <form id='form-new-vm-backup'>
+          <Wizard>
+            <Section icon='backup' title={this.props.job ? 'editVmBackup' : 'newVmBackup'}>
+              <Container>
+                <Row>
+                  <Col>
+                    <fieldset className='form-group'>
+                      <label>{_('backupOwner')}</label>
+                      <SelectSubject
+                        onChange={this.linkState('job.userId', 'id')}
+                        predicate={this._subjectPredicate}
+                        required
+                        value={this._getValue('job', 'userId', this.props.currentUser.id)}
+                      />
+                    </fieldset>
+                    <fieldset className='form-group'>
+                      <label>{_('jobTimeoutPlaceHolder')}</label>
+                      <TimeoutInput
+                        className='form-control'
+                        onChange={this.linkState('job.timeout')}
+                        value={this._getValue('job', 'timeout')}
+                      />
+                    </fieldset>
                     <fieldset className='form-group'>
-                      <label htmlFor='smartMode'>{_('smartBackupModeSelection')}</label>
+                      <label htmlFor='selectBackup'>{_('newBackupSelection')}</label>
                       <select
                         className='form-control'
-                        id='smartMode'
-                        onChange={this._handleSmartBackupMode}
+                        id='selectBackup'
+                        onChange={this.linkState('job.method')}
                         required
-                        value={smartBackupMode ? 'smart' : 'normal'}
+                        value={method}
                       >
-                        {_('normalBackup', message => <option value='normal'>{message}</option>)}
-                        {_('smartBackup', message => <option value='smart'>{message}</option>)}
+                        {_('noSelectedValue', message => <option value=''>{message}</option>)}
+                        {map(BACKUP_METHOD_TO_INFO, (info, key) =>
+                          <option key={key} value={key}>{formatMessage(messages[info.label])}</option>
```

Comment:
Original code:
```js
{map(BACKUP_METHOD_TO_INFO, (info, key) =>		
  _(info.label, message => <option key={key} value={key}>{message}</option>)		
)}
```

In the original code, the map returned an array of `FormattedMessage`s. Therefore, each element of the array didn't have a `key` prop.